### ISSUE TITLE
Publish script type generation fails

### ISSUE THREAD

**Issue by richard-cox** [rancher/dashboard#11797]
**Describe the bug**
- The shell/scripts/typegen.sh script creates typescript definitions that are bundled into the shell package
- At some point (vue3?) all commands to generate types fails

https://github.com/rancher/dashboard/actions/runs/10637724977/job/29492174914
```
Generating typescript definitions
Generating ...
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 15: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 16: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 17: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 20: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 21: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 22: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 23: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 26: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 27: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 30: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 31: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 32: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 33: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 36: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 39: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
/home/runner/work/dashboard/dashboard/shell/scripts/typegen.sh: line 40: /home/runner/work/dashboard/dashboard/shell/node_modules/.bin/tsc: No such file or directory
```

**Expected Result**
command succeeds, types created

**Comment by rak-phillip** [comment]
Has this been resolved in #12211?

@jordojordo @aalves08

**Comment by jordojordo** [comment]
@rak-phillip yes this has been fixed. Sorry I didn't realize there was a related issue.

**Comment by rak-phillip** [comment]
Closing as fixed in #12211.

Duplicate of #12223.
